charts/manticoresearch/values.yaml

```diff
@@ -201,3 +210,15 @@ persistence: {}
   ## Requires persistence.enabled: true
   ## If defined, PVC must be created manually before volume will be bound
 # existingClaim:
+
+serviceMonitor:
```

Comment:
Remove this section, it's duplicated below

Reply:
Done